### ISSUE TITLE
loss_rcnn_box is Nan

### ISSUE THREAD

**Issue by jameschartouni** [roytseng-tw/Detectron.pytorch#29]
I was able to successfully train a model with a custom dataset using the command line arguments and train.py file given. I refactored the train.py code to run with hardcoded variables instead of command line arguments. Yet in my own script, after the first step, the loss_rcnn_bbox values are Nan, which will then crash the program. What could be possible causes?

            outputs = maskRCNN(**input_data)

            rois_label = outputs['rois_label']
            cls_score = outputs['cls_score']
            bbox_pred = outputs['bbox_pred']
            loss_rpn_cls = outputs['loss_rpn_cls'].mean()
            loss_rpn_bbox = outputs['loss_rpn_bbox'].mean()
            loss_rcnn_cls = outputs['loss_rcnn_cls'].mean()
            print(outputs['loss_rcnn_bbox'].mean()) #this value is Nan
            loss_rcnn_bbox = outputs['loss_rcnn_bbox'].mean()

**Comment by JackHenry1992** [comment]
I have encountered Nan and sloved by reducing the learning rate, maybe you can try this methods.

**Comment by roytseng-tw** [comment]
Yeah , too large lr is likely to be the cause of NaN loss. BTW, it's recommend to use `train_net_step.py`, which include learning rate warm up for better stability of training.

**Comment by jameschartouni** [comment]
Your suggestions worked! Thank you!

**Comment by RakhithaRR** [comment]
Any reason as to why cls_loss and bbox_loss is always zero from the beginning of training? Is it also related to learning rate?
I'm stuck at this point. Training doesn't return the expected results due to this.
